src/util/nondet.cpp

```diff
@@ -37,21 +36,18 @@ symbol_exprt generate_nondet_int(
   const int64_t max_value,
   const std::string &name_prefix,
```

Comment:
Please document that `name_prefix` must be `id2string(function_id)` (or rename it accordingly.

Reply:
done